MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include setup.py README.rst MANIFEST.in LICENSE phabricator/interfaces.json
+include setup.py README.rst MANIFEST.in LICENSE phabricator/interfaces.json setup.cfg tox.ini
```

Comment:
We shouldn't need to include setup.cfg or tox.ini. Is there a reason these were added?